scanner/src/main/resources/runscanner.properties

```diff
@@ -1,2 +1,6 @@
 project.name:${name}
-project.version:${project.version}
\ No newline at end of file
+project.version:${project.version}
+
+# Base URL can usually be determined automatically, but may need manual setting if behind a reverse
+# proxy, depending on its configuration
+#swagger.baseUrl=
```

Comment:
This file is for holding some properties that are filled via maven-resources-plugin during the build and shouldn't contain server-specific config. There isn't a great place for this, but I guess you could put it in the webapp context xml. Just add notes to the install guide about it.